src/core/object/controller/controller.ts

```diff
@@ -317,7 +317,7 @@ export default class Controller {
 				type: 'removeFromBlocklist',
 				fn: async () => {
 					await this.blocklist.removeFromBlocklist(
-						this.connector.getChannelId?.() ?? '',
+						this.connector.getChannelInfo?.()?.id ?? '',
```

Comment:
```suggestion
						this.connector.getChannelInfo?.()?.id ?? null,
```

Wouldn't this be more correct? Why would we pass an empty string instead of null when the `id` does not exist?

Reply:
This was done because the removeFromBlocklist signature demands a string, but we can probably change that to accept null. It will not impact implementation at all.

Reply:
Done